### PR TITLE
Databases lesson2

### PR DESCRIPTION
**Pull request by maccrin**
This commit is for database2 homework

### CODE REVIEW

File: databases/week2/all_queries/task.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#Change the title of a task
+update task set title='updated-title' where id=2;
+#Change a task due date
+update task set due_date='2023-01-16' where id=2;
+#Change a task status
+update task set status_id=(select id from status where id=3) where id=2;
+#Mark a task as complete
+update task set status_id=(select id from status where id=3) where id=34;
+#Delete a task
+delete from task where id=31;
```
Comment (Elia-Git) [inline databases/week2/all_queries/task.sql:18]: Good job to use nested(sub queries)

File: databases/week2/all_queries/school_db.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+`phone` varchar(255) NOT NULL,
+`class_id` int(10) unsigned NOT NULL,
+CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `class` (`id`)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+#If you are done with the above tasks, you can continue with these advanced tasks:
+#Create an index on the name column of the student table.
+CREATE INDEX index_name ON student (name);
+#Add a new column to the class table named status which can only have the following values: not-started, ongoing, finished (hint: enumerations).
+ALTER TABLE class ADD status ENUM('not-started', 'ongoing', 'finished')NOT NULL;
```
Comment (Elia-Git) [inline databases/week2/all_queries/school_db.sql:27]: Just wow! Thx

File: databases/week2/all_queries/school_db.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
+use  school_db;
+##Create a new database containing the following tables:
+#Class: with the columns: id, name, begins (date), ends (date)
+#Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`begins` DATETIME NOT NULL,
+`ends` DATETIME NULL DEFAULT NULL
```
Comment (Elia-Git) [inline databases/week2/all_queries/school_db.sql:9]: Here data type  for columns `begins` & `ends` was supposed to be `date` not `datetime` according to the Q. Small detail

File: databases/week2/all_queries/more_queries.sql [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+join task on task.id=user_task.task_id WHERE user.email LIKE '%@spotify.com';
+#Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT user_task.task_id ,task.title FROM user_task JOIN user ON user.id=user_task.user_id
+JOIN task ON task.id= user_task.task_id JOIN status on status.id= task.status_id
+WHERE user.name ="Donald Duck" AND status.name ="Not started";
+#Get all the tasks for 'Maryrose Meadows' that were created in september
+select task.title , task.id,task.created as Month, name from user join user_task on user.id=user_task.user_id
+join task on task.id=user_task.task_id where user.name='Maryrose Meadows' and month(created)=9;
+#Find how many tasks where created in each month, e.g. how many tasks were created in october, how many tasks were created in november, etc. (hint: use group by)
+select count(id) as total_task , month(created) as Month from task group by Month;
```
Comment (Elia-Git) [inline databases/week2/all_queries/more_queries.sql:13]: Cool here too!

**Review by Elia-Git — APPROVED** [review]
Hello! 
Great staff.  
Special appreciation for the  great file structure

-    You divided your work in respective files ( even easier for anyone who reads your code).

-  One thing to small fix is please use a standard way of writing your code (either lower or upper case) in the case of using the **keywords**  https://www.w3schools.com/sql/sql_ref_keywords.asp & try to indent your code for a better appearance.

Ship it pls!

File: databases/week2/all_queries/school_db.sql (hunk 2 above)
Comment (maccrin) [reply databases/week2/all_queries/school_db.sql:27]: Thank You for review and feedback

File: databases/week2/all_queries/school_db.sql (hunk 3 above)
Comment (maccrin) [reply databases/week2/all_queries/school_db.sql:9]: Will note this,thanks.

**Pull request merged**